google-forms/src/app/pages/home/home.component.ts

```diff
@@ -68,6 +68,21 @@ export default class HomePage implements OnInit {
     this.router.navigate(['survey', id]);
   }
 
+  onDownloadForm(id: string) {
+    this.formService.downloadResponses(id).subscribe((blob: Blob) => {
+      const url = window.URL.createObjectURL(blob);
+      const a = document.createElement('a');
+      a.href = url;
+      a.download = `${id}.csv`;
+      document.body.appendChild(a);
+      a.click();
+      window.URL.revokeObjectURL(url);
+      document.body.removeChild(a);
+    }, error => {
+      console.error('Error downloading file', error);

```

Comment:
Do we not want to use the snackbar for this? Are we cool with just the console.error?